editor/project_settings_editor.cpp

```diff
@@ -220,15 +220,24 @@ void ProjectSettingsEditor::_add_feature_overrides() {
 	presets.insert("standalone");
 	presets.insert("32");
 	presets.insert("64");
-	presets.insert("Server"); // Not available as an export platform yet, so it needs to be added manually
+	presets.insert("server"); // Not available as an export platform yet, so it needs to be added manually.
+	presets.insert("linux"); // Not reported in the list of export platforms yet, so it needs to be added manually.
+	presets.insert("bsd"); // Not reported in the list of export platforms yet, so it needs to be added manually.
 
 	EditorExport *ee = EditorExport::get_singleton();
 
 	for (int i = 0; i < ee->get_export_platform_count(); i++) {
 		List<String> p;
 		ee->get_export_platform(i)->get_platform_features(&p);
 		for (List<String>::Element *E = p.front(); E; E = E->next()) {
-			presets.insert(E->get());
+			// Feature tags for platform names are lowercase.
+			if (E->get().to_lower() == "x11") {
+				// `x11` is no longer a valid feature tag name in Godot 4.0 and later.
+				// Use `linux` or `bsd` instead.
+				continue;
+			}
```

Comment:
This comes from the platform preset, no? So would `x11` even be present in the first place? I don't think this kind of compatibility code is needed here.

Reply:
I removed this part as it's no longer useful with the new approach.